client/tests/webdriver/specs/editLocation.spec.js

```diff
@@ -1,50 +1,62 @@
+import CreateNewLocation from "../pages/location/createNewLocation.page"
 import EditLocation from "../pages/location/editLocation.page"
+import ShowLocation from "../pages/location/showLocation.page"
+import { LOCATION_COORDS, LOCATION_NAME, NEW_COORDS } from "./locationUtils"
+
+describe("When editing a location", () => {
+  it("Should create a new location first", () => {
+    CreateNewLocation.open(LOCATION_NAME)
+    CreateNewLocation.nameField.setValue(LOCATION_NAME)
+    CreateNewLocation.latField.setValue(LOCATION_COORDS.lat)
+    CreateNewLocation.lngField.setValue(LOCATION_COORDS.lng)
+    CreateNewLocation.createButton.click()
+    // We are sent to showLocation page
+    ShowLocation.successMsg.waitForExist()
+    ShowLocation.successMsg.waitForDisplayed()
+  })
 
-const EX_LOCATION_ID = "f2207d9b-204b-4cb5-874d-3fe6bc6f8acd"
-
-const EXAMPLE_LAT_LNG_TO_CONVERT = {
-  lat: "47.52963",
-  lng: "-52.94612"
-}
-
-const EXAMPLE_MGRS_TO_BE_EXPECTED = "22TCT5351665858"
-
-describe("When editing an existing location", () => {
   it("Should see latitude and longitude label when selected format is LAT_LON", () => {
-    EditLocation.open(EX_LOCATION_ID)
+    ShowLocation.editButton.waitForExist()
+    ShowLocation.editButton.waitForDisplayed()
+    ShowLocation.editButton.click()
+    // Now we are in the edit page
     EditLocation.latLngLabel.waitForExist()
     EditLocation.latLngLabel.waitForDisplayed()
   })
```

Comment:
I was just thinking, we don't test that what we save is also what we get back on the edit page. Maybe you should also add a test checking that the lat/lng fields also contain the expected values.